webui/module/Restore/src/Restore/Form/RestoreForm.php

```diff
@@ -599,137 +435,95 @@ public function __construct($restore_params = null, $clients = null, $filesets =
         );
 
         // Submit button
-        if (isset($this->restore_params['client'])) {
-            $this->add(array(
-                'name' => 'form-submit',
-                'type' => 'button',
-                'options' => array(
-                    'label' => _('Restore')
-                ),
-                'attributes' => array(
-                    //'value' => _('Restore'),
-                    'id' => 'btn-form-submit',
-                    'class' => 'btn btn-primary'
-                )
-            ));
-        } else {
-            $this->add(array(
-                'name' => 'form-submit',
-                'type' => 'button',
-                'options' => array(
-                    'label' => _('Restore')
-                ),
-                'attributes' => array(
-                    //'value' => _('Restore'),
-                    'id' => 'btn-form-submit',
-                    'class' => 'btn btn-primary',
-                    'disabled' => true
-                )
-            ));
-        }
-    }
+        $form_submit = array(
+            'name' => 'form-submit',
+            'type' => 'button',
+            'options' => array(
+                'label' => _('Restore')
+            ),
+            'attributes' => array(
+                //'value' => _('Restore'),
+                'id' => 'btn-form-submit',
+                'class' => 'btn btn-primary',
+                // will be enabled by JS
+                'disabled' => true
+            )
+        );
+        $this->add($form_submit);
```

Comment:
This can probably be done with `$this->add(array(...))` like the others above.

Reply:
done